src/components/Column.jsx

```diff
@@ -13,19 +20,35 @@ const Column = ({
       </div>
       <ul className="column-content">
         {cards.map(card => (
-          <li key={card.id}>{card.title}</li>
+          <Card key={card.id} card={card}/>
         ))}
       </ul>
+      <AddButton
+        id = {`btn-${id}`}
+        name = {`btn-${id}`}
+        onSubmit ={cardTitle => createCard({title: cardTitle}, id)}
+        buttonText ="Add a card"
+        wrapperClassName="column-action-add"
+        buttonClassName = "btn-column-add"
```

Comment:
[formatting] :scissors: the spaces around the `=`, try to be consistent with your formatting. :wink: